lib/resources/r4/charge_item.yaml

```diff
@@ -100,25 +100,51 @@ fields:
     description: A defined group of bill codes that drives billing behavior.
     url: https://fhir.cerner.com/millennium/r4/financial/general/charge-item#extensions
     binding:
-      description: A defined group of bill codes that drives billing behavior. This binding is used in <extension>bill-code-schedule</extension> extensions.
+      description: A defined group of bill codes that drives billing behavior. This binding is used in <code>bill-code-schedule</code> extensions.
       terminology:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
 
+  - name: Unit Price/Net Price
+    terminology_name: .extension[x].valueMoney.currency
+    required: 'No'
+    type: Extension
+    description: Currency codes from ISO 4217 (see https://www.iso.org/iso-4217-currency-codes.html).
+    url: https://fhir.cerner.com/millennium/r4/financial/general/charge-item#extensions
+    binding:
+      description: Currency codes from ISO 4217 (see https://www.iso.org/iso-4217-currency-codes.html). This binding is used in <code>net-price</code> and <code>unit-price</code> extensions.
+      terminology:
+        - display: Currencies
+          system: urn:iso:std:iso:4217
+          info_link: http://hl7.org/fhir/r4/valueset-currencies.html
```

Comment:
make the info links https please

Reply:
All the links throughout the file have been changed.
https://github.com/cerner/fhir.cerner.com/pull/727/commits/8a15adbb97966b3f6d2a674ce02334b504011903#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR120

Reply:
i think the systems need to say http

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/727/commits/b319790b9b5eb344bc6e6e4c366e62244e89f590#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR132

Reply:
Added s for all the systems: https://github.com/cerner/fhir.cerner.com/pull/727/commits/8476a1709f480d612f31565ff93d84ac2b89285c#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR132

Reply:
Done as requested :https://github.com/cerner/fhir.cerner.com/pull/727/commits/2bd59ce3637aac550ccedcf571b7e2dc289a5bc1#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR145

https://github.com/cerner/fhir.cerner.com/pull/727/commits/2bd59ce3637aac550ccedcf571b7e2dc289a5bc1#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR166